### PR TITLE
Added Url-shortener

### PR DESCRIPTION
**Pull request by Aditijainnn**
Closes: 

- **Title** : Url shortener
- **Name:** :  Aditi Jain
- **Idenitfy yourself:** :  GSSOC

### Describe the add-ons or changes you've made 📃
Added url shortener in next js project

### Screenshots 📷

image

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by Aditijainnn** [comment]
@Avdhesh-Varshney  please merge my pr

File: Next-JS-Projects/Intermediate/Url-Shortener/Screenshot.webp.png
Comment (Avdhesh-Varshney) [inline Next-JS-Projects/Intermediate/Url-Shortener/Screenshot.webp.png:1]: It should be in `webp` format and named as `screenshot` => `screenshot.webp` ok

File: Next-JS-Projects/Intermediate/Url-Shortener/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+# or
+pnpm dev
+```
+
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+
+
+
+## :zap: Screenshots 📸
+<img src="./screenshot.webp.png"/>
```
Comment (Avdhesh-Varshney) [inline Next-JS-Projects/Intermediate/Url-Shortener/README.md:45]: Don't forget to update it.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
You've not linked your issue with pr.

**Pull request closed without merging**